terraform/openstack/hosts-floating/main.tf

```diff
@@ -46,6 +74,7 @@ resource "openstack_compute_instance_v2" "resource" {
   image_name            = "${ var.image_name }"
   flavor_name           = "${ var.resource_flavor_name }"
   security_groups       = [ "${ var.security_groups }" ]
+  scheduler_hints = { group = "${ openstack_compute_servergroup_v2.resource-sg.id }" }
```

Comment:
Do the worker nodes really need anti-affinity? I imagine that could be a problem in saturated OpenStack installations?